Cargo.toml

```diff
@@ -41,12 +41,12 @@ picky-asn1-x509 = { version = "0.4.0", optional = true }
 users = "0.11.0"
 libc = "0.2.86"
 anyhow = "1.0.38"
-rust-cryptoauthlib = { version = "0.3.0", optional = true }
+rust-cryptoauthlib = { version = "0.3.1", optional = true }
 prost = { version = "0.7.0", optional = true }
 
 [dev-dependencies]
 rand = { version = "0.8.3", features = ["small_rng"] }
-rust-cryptoauthlib = { version = "0.3.0", features=["software-backend"]}
+rust-cryptoauthlib = { version = "0.3.1", features=["software-backend"]}
```

Comment:
Just my curiosity but is this needed if `rust-cryptoauthlib` is already a normal dependency?

Reply:
We assume the version 0.3.1 is the first supporting both scenarios of exporting public key (first from a slot with a private key, second directly from the slot with public key). Assuming that after 0.3.1 became an official crate it started being preferred anyway, this version bump is probably overzealous.

Reply:
Ah sorry I realized my sentence was not clear. I meant: is it needed to have `rust-cryptoauthlib` written as a dependency in both `[dependencies]` and `[dev-dependencies]` sections of the `Cargo.toml` file?

Reply:
Ah right I missed that! Thanks for the explanation, that makes sense